src/routes/posts.tsx

```diff
@@ -45,16 +46,30 @@ const config = {
 export function Posts(): ReactElement | ReactElement[] | null {
   const posts = usePosts();
   if (posts.isLoading) {
```

Comment:
Try adding empty lines between consts, improved the readability. Not every consts, for example a list of assignments is fine:

```
const a = 'a';
const b = 'b';
const c = 'c';
```

but if there's a condition, some method and etc, add an empty line:

```
const a = 'a';
const b = 'b';
/* empty line here */
if (...) {}
```

Reply:
Relevant to all files.